R/Steph_Function_Attempts.R

```diff
@@ -0,0 +1,101 @@
+##This is my script for writing a function to edit an attribute in an existing attributes table.
+
+library(testthat)
```

Comment:
Make sure you're only defining libraries that you actually use here - are you using testthat?

Reply:
This library is required for creating the dummy package, I believe. But is no longer necessary to include since I have that package's PIDs.

Reply:
Actually, `create_dummy_package` is part of the `arcticdatautils` library. We use `testthat` for our unit tests. No problem, though!